omnigibson/prims/entity_prim.py

```diff
@@ -1032,7 +1034,8 @@ def joint_damping(self):
         """
         return np.concatenate([joint.damping for joint in self._joints.values()])
 
-    @property
+    # TODO: These are cached, but they are not updated when the joint limit is changed
```

Comment:
not sure if I understand this TODO.

Reply:
I think this means that when the joint limit is updated in the USD, this property will not automatically know to clear its cache and update during the next call